### PR TITLE
Move all scoreboard related configuration options into a config.json file

### PR DESCRIPTION
**Pull request by swemoney**
Move the commandline arguments that control the scoreboard into a config.json file. led-matrix arguments remain as command line arguments. Also added the config.json to .gitignore.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -35,4 +38,4 @@
     else:
       # The mlbgame API returns a 2D array with the list of games as the first index,
       # hence the 'games[0]'
-      GameRenderer(matrix, canvas, games[0], args).render()
+      GameRenderer(matrix, canvas, games[0], args, config).render()
```
Comment (ajbowler) [inline main.py]: Can probably just remove the args param now

File: utils.py [hunk 2]
```diff
@@ -76,3 +70,18 @@ def led_matrix_options(args):
     options.disable_hardware_pulsing = True
 
   return options
+
+class ScoreboardConfig:
```
Comment (ajbowler) [inline utils.py]: Move to separate file? Not a big fan personally of having this in utils

**Comment by ajbowler** [comment]
Can you update the README? If not I can do it later today

**Comment by swemoney** [comment]
Should be all set with those changes. Let me know what you think.

**Pull request closed without merging**